.github/workflows/linux-workflow.yml

```diff
@@ -31,23 +31,23 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - os: ubuntu-20.04
+          - os: ubuntu-18.04
```

Comment:
Hmm, why the downgrade in ubuntu?  20.04 is the current LTS, and it would be nice to continue using LTS as a general build target.  If 18.04 is truly needed for AppImage generation, this seems a little heavy-handed and could perhaps be contained to just the relevant step.